tests/unit/flipswitch/index.html

```diff
@@ -15,11 +15,13 @@
 	<link rel="stylesheet" href="../../../external/qunit/qunit.css"/>
 	<link rel="stylesheet" href="../../jqm-tests.css"/>
 	<script src="../../../external/qunit/qunit.js"></script>
+	<script src="../../../external/qunit-assert-classes/qunit-assert-classes.js"></script>
 	<script>
 		$.testHelper.asyncLoad([
 			[ "widgets/page" ],
 		[
 				"widgets/forms/flipswitch",
+				"widgets/forms/flipswitch.backcompat"
```

Comment:
This file should set `$.ui.backcompat = false` and not include the backcompat extension. Any tests that need the backcompat should be moved to a separate file that is a copy of this file except it includes backcompat extension and tests.